DEPS

```diff
@@ -18,6 +18,10 @@ deps = {
   "vendor/bat-native-tweetnacl": "https://github.com/brave-intl/bat-native-tweetnacl.git@1b4362968c8f22720bfb75af6f506d4ecc0f3116",
   "components/brave_sync/extension/brave-sync": "https://github.com/brave/sync.git@e32fd5828b63b053c427c48ec5139504a9c4e9f2",
   "components/brave_sync/extension/brave-crypto": "https://github.com/brave/crypto@c07866c3b9c8264195756a81112bb74b8fa90858",
+  "vendor/bat-native-ads": "https://github.com/brave-intl/bat-native-ads.git@635cd13137b87407c97a0a825c78d941c99228cb",
+  "vendor/bat-native-usermodel": "https://github.com/brave-intl/bat-native-usermodel.git@3eb3883b0d30cbe4d6f1d907d6c76f28728eb17c",
+  "vendor/bat-native-confirmations": "https://github.com/brave-intl/bat-native-confirmations.git@9410bd8306aacf11cadddde4bb17e9a79cf29d4e",
+  "vendor/challenge-bypass-ristretto-ffi/src": "https://github.com/brave-intl/challenge-bypass-ristretto-ffi@6f33c811ef682a9c23b0134dcd2aa8b478c55231",
```

Comment:
@riastradh-brave please do a quick sec review of these deps. thanks

Reply:
I think review for `bat-native-confirmations` and `challenge-bypass-ristretto-ffi` can / should be decoupled from this PR. they are not hooked up - maybe we should remove them from the deps in this PR?

Reply:
agree, plz remove if these aren't used and don't require review yet

Reply:
Discussed with @bridiver and he is going to remove, thanks